move_helper.js

```
@@ -0,0 +1,97 @@
+  consumer_secret: config.get("twitter.consumer_secret"),
+  access_token: config.get("twitter.access_token"),
+  access_token_secret: config.get("twitter.access_token_secret"),
+  timeout_ms: 60 * 1000, // optional HTTP request timeout to apply to all requests.
+  strictSSL: true // optional - requires SSL certificates to be valid.
+});
+
+async function main(){
+  let accs = await getAccounts();
+  logger.info(`Trying to move ${accs.length} accounts...`)
+  for (let i in accs){
+    try {
+      //Get user details from twitter.
+      let data = await T.get('users/show', { screen_name: accs[i] });
+      //Create a account for the user by id.
+      let usr = data.data.id_str;
+      await getAddress(id(usr));
+      //Move over from old account to the new account
+      const balanceFromOld = await lbry.getBalance(`twttr-${accs[i]}`);
+      if (balanceFromOld !== 0) {
+        let res = await lbry.move(
+          `twttr-${accs[i]}`,
+          id(usr),
+          Number(balanceFromOld)
+        );
+        // If move is successful, log it!
+        if (res) logger.info(`Transferred ${balanceFromOld} LBC from twttr-${accs[i]} to ${id(usr)}!`);
+      }
+    }catch(e){
+      logger.info(`Could not sync ${accs[i]}, error occured:`, e.allErrors);
+      notSynced.push({ user: accs[i], error: e.allErrors});
+      logger.info("Could not sync these:"+JSON.stringify(notSynced));
+    }
+  }
+}
+// Get a list of all twitter accounts on lbrycrd.
+async function getAccounts(){
+  let accs = await lbry.listAccounts();
+  accs = Object.entries(accs);
+  let accsArr = [];
+  for (let i in accs){
+    if(accs[i][0].startsWith('twttr-')) accsArr.push(accs[i][0].substring(6));
+  }
+  return accsArr;
+}
+
+async function getAddress(userId) {
+  try {
+    let uAddresses = await lbry.getAddressesByAccount(userId);
+    if (uAddresses.length > 0) return;
+    await lbry.getNewAddress(userId);
+    return;
+  } catch (e) {
+    logger.error(e);
```

you're catching the error here, but what happens if something doesn't go right? Will a tip go through and fail along the process somewhere?

---

It should probably not be catched there, instead it should throw an error in the main loop, making it impossible to try and send coins over to a non existing account